test/tmp-fixtures.js

```diff
@@ -0,0 +1,194 @@
+module.exports = {
```

Comment:
Should this be dropped now that test/fixtures/json-specs/wildcard_matcher_tests.json is available?

Reply:
🤦 👍